src/app/organization/organization.service.ts

```diff
@@ -330,7 +330,9 @@ export class OrganizationService {
    * @returns {Observable<any>} SMS Campaigns data
    */
   getSmsCampaigns(): Observable<any> {
-    return this.http.get('/smscampaigns');
+    let countryId = JSON.parse(sessionStorage.getItem('selectedCountry'))?.id;
```

Comment:
I think maybe a small wrapper service to get/set these session variables could be useful, to provide some error handling...

Reply:
Good catch. Applied the suggestion